ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/metadata.schema.yml

```diff
@@ -180,3 +180,46 @@ keys:
                               type: str
                             preference:
                               type: str
+          internet_exit_policies:
+            type: list
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                type:
+                  type: str
+                city:
+                  type: str
+                country:
+                  type: str
+                upload_bandwidth:
+                  type: int
+                download_bandwidth:
+                  type: int
+                firewall:
+                  type: bool
```

Comment:
default in the code (False)

Reply:
Not really. The schema describes the data model between eos_designs and cv_deploy/cv_workflow. The default values are implemented inside eos_designs. I have updated the schema with all the relevant values being required.